dali/kernels/alloc.cc

```diff
@@ -79,20 +83,62 @@ struct Allocator<AllocType::Unified> {
 };
 
 void *Allocate(AllocType type, size_t size) noexcept {
+#if !defined(__AARCH64_GNU__) && !defined(__AARCH64_QNX__)
   VALUE_SWITCH(type, type_label,
     (AllocType::Host, AllocType::Pinned, AllocType::GPU, AllocType::Unified),
     (return Allocator<type_label>::Allocate(size)),
     (assert(!"Invalid allocation type requested");
     return nullptr;)
   );  // NOLINT
+#else
+    void *ptr = nullptr;
+    switch (type) {
+    case AllocType::Host:
+           ptr = Allocator<AllocType::Host>::Allocate(size);
```

Comment:
Code formatting is inconsistent with the rest of project.

Reply:
I will fix this

Reply:
fixed